tf_encrypted/protocol/securenn/securenn.py

```diff
@@ -353,9 +353,38 @@ def relu(self, x):
 
     :param PondTensor x: Input tensor.
     """
-    with tf.name_scope('relu'):
-      drelu = self.non_negative(x)
-      return drelu * x
+
+    def relu(x):
+      with tf.name_scope('relu'):
+        drelu = self.non_negative(x)
+        return drelu * x
+
+
+    max_size = 224*224*95
```

Comment:
this could be an optional parameter for `relu`, allowing experts to tweak as needed

Reply:
done

Reply:
@mortendahl this caused the CI the crash because now the secureNN relu params differ then pond's relu. This can be solved by adding this to pond as well, even though it will not be used there..

Reply:
what if we put it in the kwargs instead and add that to both? i'm thinking that this is a securenn specific implementation detail that we may not want to have as part of the public API.